config/ups.php

```diff
@@ -14,4 +14,5 @@
     'access_key' => env('UPS_ACCESS_KEY', 'test'),
     'user_id'    => env('UPS_USER_ID', 'test'),
     'password'   => env('UPS_PASSWORD', 'test'),
+    'sandbox'   => env('UPS_SANDBOX', 'true') == 'true',
```

Comment:
When you get it from Laravel helper it'll be automatically converted. See https://github.com/laravel/framework/blob/5.3/src/Illuminate/Foundation/helpers.php#L428